packages/builder/src/components/backend/DatasourceNavigator/TableIntegrationMenu/IntegrationConfigForm.svelte

```diff
@@ -97,6 +123,45 @@
             error={$validation.errors[configKey]}
           />
         </div>
+      {:else if schema[configKey].type === "fieldGroup"}
+        <div class="spectrum-Accordion" role={configKey}>
+          <div class="spectrum-Accordion-item {openAccordionItems[configKey]}">
+            <h3 class="spectrum-Accordion-itemHeading">
+              <button
+                class="spectrum-Accordion-itemHeader"
+                type="button"
+                on:click={() =>
+                  (openAccordionItems[configKey] =
+                    openAccordionItems[configKey] !== "is-open"
+                      ? "is-open"
+                      : "")}
+              >
+                {getDisplayName(configKey)}
+              </button>
+              <svg
+                class="spectrum-Icon spectrum-UIIcon-ChevronRight100 spectrum-Accordion-itemIndicator"
+                focusable="false"
+                aria-hidden="true"
+              >
+                <use xlink:href="#spectrum-css-icon-Chevron100" />
+              </svg>
+            </h3>
+            <div class="spectrum-Accordion-itemContent" role={configKey}>
```

Comment:
This should probably be a component in BBUI - rather than configured in the builder.

Reply:
Moved into BBUI!